tasks/SticklerForRules.tex

```diff
@@ -0,0 +1,46 @@
+\section{Stickler for the Rules [Party Host]}
+The robot has to enforce house rules set by the homeowner.
+
+
+\subsection{Main Goal}
+The robot has to recognize when party guests are breaking rules and politely ask them to stop.
```

Comment:
This is somehow ambiguous to me. Traditionally in @Home recognize means _(Person, Name)_ pairing. My suggestion is:

_Identify and detain all guests breaking the [house] rules_